src/agent/src/sandbox.rs

```diff
@@ -181,7 +181,6 @@ impl Sandbox {
         Ok(true)
     }
 
-    #[instrument]
     pub fn add_container(&mut self, c: LinuxContainer) {
```

Comment:
Does instrumenting `add_container`, `get_container`, and `find_process` produce a lot of output? I agree that they are simple enough that we don't need to trace them, but I'm wondering if having a few isolated spans in other cases is really a problem.

Reply:
Precisely to say is that `get_container` generates too many spans. RPC calls `write_stdin`/`read_stdout`/`read_stderr` now don't generate any spans, this calls all call `get_container`.

Isolated spans are not an important problem(or bug), they only are split from their call hierarchy.